pyramid_oereb/standard/sources/plr.py

```diff
@@ -472,11 +472,14 @@ def read(self, real_estate, bbox):
                     if len(geometry_results) == 0:
                         self.records = [EmptyPlrRecord(self._theme_record)]
                     else:
+                        relevant_plrs = {}
                         self.records = []
                         for geometry_result in geometry_results:
-                            self.records.append(
-                                self.from_db_to_plr_record(geometry_result.public_law_restriction)
-                            )
+                            plr = geometry_result.public_law_restriction
+                            plr_key = '{}_{}'.format(self._theme_record.code, plr.type_code)
```

Comment:
I'm note sure about that. Perhaps it's not safe but it's safe enough for the test.